src/placement/jump_map.c

```diff
@@ -256,14 +254,15 @@ get_num_domains(struct pool_domain *curr_dom, uint32_t allow_status, pool_comp_t
 	else
 		num_dom = curr_dom->do_child_nr;
 
-	if (allow_status & PO_COMP_ST_NEW)
+	if (allow_status & PO_COMP_ST_UP)
 		return num_dom;
 
 	if (curr_dom->do_children == NULL || curr_dom->do_comp.co_type == fdom_lvl) {
 		next_target = &curr_dom->do_targets[num_dom - 1];
 		status = next_target->ta_comp.co_status;
 
-		while (num_dom - 1 > 0 && status == PO_COMP_ST_NEW) {
+		while (num_dom - 1 > 0 && status == PO_COMP_ST_UP &&
+		       next_target->ta_comp.co_fseq == 1) {
```

Comment:
just confirm that "status == PO_COMP_ST_UP && next_target->ta_comp.co_fseq == 1" only possible for the case of POOL_EXTEND (you added code in this PR) right?
And for the other case of "status == PO_COMP_ST_UP" that is POOL_REINT, that case's co_fseq should not equal to 1?

Reply:
>just confirm that "status == PO_COMP_ST_UP && next_target->ta_comp.co_fseq == 1" only possible for the case of POOL_EXTEND (you added code in this PR) right?

right.

>And for the other case of "status == PO_COMP_ST_UP" that is POOL_REINT, that case's co_fseq should not equal to 1?

correct, since reintegration can only be done from excluded(DOWNOUT) target, so co_fseq must > 1, since its initial value is 1 anyway.